tests/modules/mod.rs

```diff
@@ -674,7 +674,7 @@ fn module_dir_import_twice_no_panic() {
 fn not_allowed_submodule_file() {
```

Comment:
This test should be either removed, or renamed (incl. the sample files) from “not allowed” to “allowed”.

Reply:
i removed it in fc203d1b3 :ok_hand: